db/install.xml

```diff
@@ -15,5 +15,28 @@
         <KEY NAME="questionid" TYPE="foreign" FIELDS="questionid" REFTABLE="question" REFFIELDS="id" />
       </KEYS>
     </TABLE>
+    <TABLE NAME="question_package_questionpy" COMMENT="Available QuestionPy Packages on the Moodle Server">
```

Comment:
Ich glaube, dass wir in den Tabellennamen einen Fehler gemacht haben. Die Namen sollen eigentlich mit dem Frankenstyle beginnen, d.h. bei uns `qtype_questionpy` und diese Tabelle sollte dann `qtype_questionpy_packages` heißen.
Wir müssten dann die andere Tabelle auch mal bei Gelegenheit umbenennen.

Reply:
anscheinend sind beide prefixe ok. also question_ und qtype_